scripts/split-state.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+## Split near/res/state into several at most 40M files to fit github
+split -b 40000000 near/res/testnet_records.json near/res/testnet_records.json.
```

Comment:
40M also works hee\re, would be more readable. Also adding `near/res/testnet_records.json` to .gitignore?

Reply:
From @bowenwang1996 it can be actually 90M here, github allows at most 100M.